rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-disable-ssh-azure",
+        "description": "M1042 - Disable or Remove Feature or Program - Disable the remote service (ex: SSH, RDP, etc.) in azure if it is unnecessary.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND azure_security_group THAT ALLOWS AS rule Internet WHERE rule.inbound=true AND rule.fromPort <= 22 AND rule.toPort >= 22",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-disable-rdp-aws",
+        "description": "M1042 - Disable or Remove Feature or Program - Disable the remote service (ex: SSH, RDP, etc.) in aws if it is unnecessary.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND aws_security_group AS sg THAT ALLOWS AS rule Internet WHERE rule.ingress=true AND rule.fromPort <= 3389 AND rule.toPort >= 3389",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-disable-rdp-gcp",
+        "description": "M1042 - Disable or Remove Feature or Program - Disable the remote service (ex: SSH, RDP, etc.) in google cloud if it is unnecessary.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Internet THAT ALLOWS AS rule google_compute_firewall AS firewall THAT PROTECTS google_compute_network AS network THAT CONTAINS google_compute_subnetwork AS subnetwork WHERE firewall.ingress=true AND rule.ipProtocol='rdp' AND rule.fromPort <= 3389 AND rule.toPort >= 3389",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-disable-rdp-azure",
+        "description": "M1042 - Disable or Remove Feature or Program - Disable the remote service (ex: SSH, RDP, etc.) in azure if it is unnecessary.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND azure_security_group THAT ALLOWS AS rule Internet WHERE rule.inbound=true AND rule.fromPort <= 3389 AND rule.toPort >= 3389",
+            "version": "v1"
+          }
+        ],
+        "alertLevel": "HIGH"
+      },
+      {
+        "name": "remote-services-session-hijacking-block-unecessary-traffic",
+        "description": "M1030 - Network Segmentation - Enable firewall rules to block unnecessary traffic between network security zones within a network.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND Firewall WITH ingressRules = 'null' OR egressRules = 'null'",
```

Comment:
would this just return unused/unconfigured firewalls or security groups? those without any in/egress rules defined?

Reply:
`Enable firewall rules to block unnecessary traffic between network security zones within a network. `
Changed query to:
FIND Firewall WITH ingressRules !~= /CidrIp/i AND egressRules !~= /CidrIp/i
This will look for firewalls in default or unconfigured form.